Porting old Umbraco API Controller [Debugging > Logging] (#6513)
umbraco/UmbracoDocs: 14/umbraco-cms/fundamentals/code/debugging/logging.md

2021-08-10 09:33:23,677 [P25776/D1/T22] INFO   Umbraco.Cms.Core.Services.Implement.ContentService - Document Home (id=1062) has been published.
```

> Here is an example of the same log message represented as JSON. You can see that we have much more information. This would allow us to search and filter logs based on these properties with an appropriate logging system.

Reviewer: Attempted a rewrite here, to get rid of the "You can see" part, which we should try to avoid ⭐ 
What do you think? 🤔 

```suggestion
Here is an example of the same log message represented as JSON. More information is available and allows you to search and filter logs based on these properties with an appropriate logging system.
```
